apps/web/src/components/atoms/button/Button.tsx

```diff
@@ -34,6 +38,8 @@ export const Button: React.FC<React.PropsWithChildren<ButtonProps>> = ({
         styles.button,
         {
           [styles.primary]: primary,
+          [styles.outlined]: outlined,
+          [styles.signIn]: signIn,
```

Comment:
nie wiem czy signIn tutaj to dobry pomysł, przycisk nie powinien wiedzieć że jest tam używany

Może lepiej dodać go jako komponent w katalogu auth/signin